### PR TITLE
Feature/issue18 ladders improvment into develop

### PR DESCRIPTION
**Pull request by donpablom**

### CODE REVIEW

File: libraries/mainMethods.js [hunk 1]
```diff
@@ -178,34 +178,94 @@ async function sendGamesReport(gamesroom, members) {
...
-    var embedData = new EmbedBuilder()
-        .setColor('#0099ff')
-        .setTimestamp()
-        .setFooter({ text: 'AOE4 Companion', iconURL: 'https://i.imgur.com/AfFp7pu.png' })
-        .setTitle(ladderName);
+    let ladderName = guildData.name + ' ' + ladderType;
+    let embedsArray = [];
+
+    // for testing purposes
+    playerData = addGeneratedTestData(playerData);
```
Comment (jascaldeira) [inline libraries/mainMethods.js:190]: should this be here?

File: commands/contributors.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ module.exports = {
             .setFooter({ text: 'Please support the project using the following URL: https://github.com/jascaldeira/AOE4-DISCORD-BOT', iconURL: 'https://i.imgur.com/AfFp7pu.png' });
         embedData.setTitle('Contributors');
         embedData.addFields(
-            { name: '- ', value: '- Caldeira\n- PM303' }
+            { name: '- ', value: '- Caldeira\n- PM303\n- donPabloM' }
```
Comment (jascaldeira) [inline commands/contributors.js:15]: just if you want, you can add yourself in the Readme file as well.

**Review by jascaldeira — CHANGES_REQUESTED** [review]

File: libraries/mainMethods.js (hunk 1 above)
Comment (donpablom) [reply libraries/mainMethods.js:190]: honestly, it is not used, not needed for proper work of code just for dev purouses

File: commands/contributors.js (hunk 2 above)
Comment (donpablom) [reply commands/contributors.js:15]: nice, thanks

File: libraries/mainMethods.js (hunk 1 above)
Comment (donpablom) [reply libraries/mainMethods.js:190]: done

**Pull request merged**